src/main/java/com/example/moviemanager/service/model/api/MovieDetailsFromApi.java

```diff
@@ -0,0 +1,7 @@
+package com.example.moviemanager.service.model.api;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record MovieDetailsFromApi(String overview, double vote_average) {
```

Comment:
You shouldn't emphasize that details are from api. Simply MovieDetails would be enough in my opinion